File: ArrayAndList/remove_duplicate_array/remove_duplicates.py
arr=[1,1,2,2,3,3,4,5]
n=len(arr)

res=[]
size=0

for i in range(n):
    found=False
    for j in range(len(res)):
        if(arr[i]==res[j]):
            found=True
            break
    if not found:
         res.append(arr[i])
        
print(res)
       

File: ArrayAndList/remove_duplicate_array/remove_duplicates.js
let arr=[1,2,3,2,1,1,4,4,5];
let n=arr.length;

let res=[];
let size=0;

for(let i=0;i<n;i++){
    let found=false;
    for(let j=0;j<size;j++){
        if(arr[i]===res[j]){
            found =true;
            break;
        }
    }
    if(!found){
        res[size]=arr[i];
        size++;
    }


}
for(let i=0;i<size;i++){
    console.log(res[i]);
}
